GitUI/UserControls/BlameControl.cs

```diff
@@ -90,6 +90,8 @@ public void UpdateShowLineNumbers()
             BlameAuthor.ShowLineNumbers = AppSettings.BlameShowLineNumbers;
         }
 
+        public void GoToLine(int line) => BlameFile.GoToLine(line);
```

Comment:
This is not used by this PR. Will it be used in a follow-up?

Reply:
It was in a previous version and may be in a future but as it is not used right now it will be removed.
As the viewing of files is done async, the line to go to should be provided when viewing the file. 
This is why `feature/i9796-blame-line-to-diff` is so complex.